core/kazoo_ast/src/cb_api_endpoints.erl

```diff
@@ -31,12 +33,18 @@
 -spec to_ref_doc() -> 'ok'.
 -spec to_ref_doc(atom()) -> 'ok'.
 to_ref_doc() ->
+    ensure_ref_dir(),
     lists:foreach(fun api_to_ref_doc/1, ?MODULE:get()).
 
 to_ref_doc(CBModule) ->
+    ensure_ref_dir(),
     Path = code:which(CBModule),
     api_to_ref_doc(hd(process_module(Path, []))).
 
+-spec ensure_ref_dir() -> 'ok'.
+ensure_ref_dir() ->
+    'ok' = filelib:ensure_dir(filename:join([?REF_PATH, ".placeholder"])).
```

Comment:
> could be someone built a release and wants docs for some reason?

that doesn't make sense.

> Doesn't hurt to have it

lots of things can go by that comment... I thought we were not supposed to bring in dead code?